src/main/webapp/js/playGame.js

```diff
@@ -55,10 +56,7 @@ function initMapToPlayGame() {
     // puts the user in streetView
     panorama.setVisible(true);
 
-    const minimapControlDiv = document.createElement("div");
-    addMinimap(minimapControlDiv, panorama);
-    minimapControlDiv.index = 1;
-    panorama.controls[google.maps.ControlPosition.LEFT_BOTTOM].push(minimapControlDiv);
+    addMinimap(panorama);
```

Comment:
Instead of having it global can u have addMiniMap return the minimap and use it that way?

Reply:
Minimap is no longer global, thanks.